homeassistant/components/logger/__init__.py

```diff
@@ -77,6 +79,23 @@ def update_log_filter(self, logs):
         self._log_rx = re.compile("".join(["^(?:", "|".join(names_by_len), ")"]))
         self._logs = logs
 
+    def set_logger_level(self):
```

Comment:
You know thinking about this whole implementation. Is there any reason why we can't just change the levels on the original log objects themselves instead of having to do this ourselves?

```python
logging.getLogger(name).setLevel(level)
```

Reply:
In theory that should work https://github.com/python/cpython/blob/3.8/Lib/logging/__init__.py#L1663

Need to see how it will behave with loggers that get created later.

Reply:
Will do some testing with this.

Reply:
#37780

Reply:
Nice. Will do testing with it tomorrow.